src/R/Components/Test/MefCompositionTests.cs

```diff
@@ -56,9 +56,5 @@ public class MefCompositionTests : IDisposable {
         public void InteractiveWindowFactoryService() {
             _exportProvider.GetExportedValue<IInteractiveWindowFactoryService>().Should().NotBeNull();
         }
-
-        public void Dispose() {
-            (_exportProvider as IDisposable)?.Dispose();
```

Comment:
Disposing `ExportProvider` helps detecting static caches of MEF components, which should be considered an anti-pattern (MEF can be forced to rebuild its instances, while static cache will have no idea about it).